docs/marketplace/user/features/202106.0/merchant-products.md

```diff
@@ -0,0 +1,34 @@
+Merchant Products are the products that a Merchant owns. Besides creating the offers for the products that the Marketplace Administrator suggests, a Merchant can create their own unique products. These products possess the same characteristics the usual abstract and concrete products have, but in addition, every such product has Merchant-related information such as merchant reference. Merchants manage stock, set prices 
```

Comment:
offers for the products - of other Marchants - not only Marketplace Administrator

Reply:
will add